lib/krane/template_sets.rb

```diff
@@ -37,23 +37,30 @@ def ejson_secrets_file
         end
       end
 
-      def validate
+      def validate(render_erb: true)
         errors = []
+        supported_extensions = if render_erb
+          TemplateSets::VALID_TEMPLATES
+        else
+          TemplateSets::VALID_TEMPLATES.reject { |extension| extension.include?('erb') }
+        end
+
         if Dir.entries(@template_dir).none? do |filename|
-             filename.end_with?(*TemplateSets::VALID_TEMPLATES) ||
+             filename.end_with?(*supported_extensions) ||
              filename.end_with?(EjsonSecretProvisioner::EJSON_SECRETS_FILE)
            end
-          return errors << "Template directory #{@template_dir} does not contain any valid templates"
+          return errors << "Template directory #{@template_dir} does not contain any valid templates (supported " \
+              "suffixes: #{supported_extensions.join(', ')}, or #{EjsonSecretProvisioner::EJSON_SECRETS_FILE})"
         end
 
         @files.each do |filename|
           filename = File.join(@template_dir, filename)
           if !File.exist?(filename)
             errors << "File #{filename} does not exist"
-          elsif !filename.end_with?(*TemplateSets::VALID_TEMPLATES) &&
+          elsif !filename.end_with?(*supported_extensions) &&
                 !filename.end_with?(EjsonSecretProvisioner::EJSON_SECRETS_FILE)
             errors << "File #{filename} does not have valid suffix (supported suffixes: " \
-              "#{TemplateSets::VALID_TEMPLATES.join(', ')}, or #{EjsonSecretProvisioner::EJSON_SECRETS_FILE})"
+              "#{supported_extensions.join(', ')}, or #{EjsonSecretProvisioner::EJSON_SECRETS_FILE})"
           end
```

Comment:
What do you think about explicitly handling the case when `render_erb: false` and we find `.erb` files? This would be a good place to give a hint to use `krane render | krane deploy`